k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/ManagedDeliveryK8sPlugin.kt

```diff
@@ -1,9 +1,6 @@
 package com.amazon.spinnaker.keel.k8s
 
-import com.amazon.spinnaker.keel.k8s.resolver.DockerImageResolver
-import com.amazon.spinnaker.keel.k8s.resolver.HelmResourceHandler
-import com.amazon.spinnaker.keel.k8s.resolver.K8sResolver
-import com.amazon.spinnaker.keel.k8s.resolver.K8sResourceHandler
+import com.amazon.spinnaker.keel.k8s.resolver.*
```

Comment:
not sure if this was done by intelliJ or you, but I prefer defining imports explicitly. more of a golang pattern.

Reply:
Yeah I let IntelliJ auto format. I'll see if I can disable it in IntelliJ